scope.md

```diff
@@ -91,6 +91,51 @@ priority:
 * QR-code triggered payment. User points phone at QR code which represents a Web Page. In vanilla mode, Web Page includes "Buy" button to trigger Payment Request. But in streamlined mode, SPC transaction dialog is immediately displayed to the user for a default instrument associated with the payment method.
 * Tap-to-pay (NFC)
 
+## User Stories
+
+### Out of band enrollment
+
+* While mobile banking, Alice enroll her authenticator to be used with all her credit cards issued by this bank.
+* A few days later during checkout, Alice is prompted to confirm a transaction with one of those cards via her authenticator.
+
+### In-transaction enrollment, authentication same merchant
+
+* While checking out, Alice selects a payment instrument and is authenticated before being able to use it.
```

Comment:
Brief note: authenticated via *other means* than SPC (e.g. OTP).

Reply:
I've tweaked the text accordingly.